mscore/masterpalette.cpp

```diff
@@ -203,9 +204,9 @@ MasterPalette::MasterPalette(QWidget* parent)
 
 void MasterPalette::retranslate(bool firstTime)
       {
-      keyItem->setText(0, tr("Key Signatures"));
-      timeItem->setText(0, tr("Time Signatures"));
-      symbolItem->setText(0, tr("Symbols"));
+      keyItem->setText(0, qApp->translate("Palette", "Key Signatures"));
+      timeItem->setText(0, qApp->translate("Palette", "Time Signatures"));
+      symbolItem->setText(0, qApp->translate("MasterPalette", "Symbols"));
```

Comment:
I think this line refers to a meanwhile removed string from masterpalette.ui since e6997f3e, so it does translate using the current translations fir 2.0, but won't once we re-sync Transifex with the current strings?